### DISCUSSION TITLE
migrate Foreign key constraint is incorrectly formed

### DISCUSSION THREAD

**Discussion by eeviefogarty** [prisma/prisma#13554] [Q&A]
```
Error: P3018

A migration failed to apply. New migrations cannot be applied before the error is recovered from. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

Migration name: 20220529215906_meouwuas

Database error code: 1005

Database error:
Can't create table `krackin`.`config` (errno: 150 "Foreign key constraint is incorrectly formed")

Please check the query number 4 from the migration file.
```

![image](https://user-images.githubusercontent.com/89327699/170891268-ebf846d1-69db-4880-90d6-f73825e89516.png)
![image](https://user-images.githubusercontent.com/89327699/170891290-6cbfa7a6-616c-4e7f-8ab4-0220cea04c13.png)

**Comment by eeviefogarty** [comment]
![image](https://user-images.githubusercontent.com/89327699/170891312-eff29de6-2673-43ba-8249-8ab9bf55674d.png)

**Reply by janpio** [reply]
Can you share the full SQL of the generated migration please?
Can you please also share the actual text instead of just screenshots - super hard to execute the SQL from a picture without bein unsure that one did not add a typo. Thanks.

**Reply by vladi-stevanovic** [reply]
👋 Hello @ian-ok  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
